### PR TITLE
feat(Shared): Migrate Scrollview

### PR DESCRIPTION
**Pull request by kevinvangelder**

### CODE REVIEW

**Comment by rozele** [comment]
Looks like you have a commit that updated the examples submodule.

File: ReactWindows/ReactNative.Net46/Bridge/DispatcherHelpers.cs [hunk 1]
```diff
@@ -54,7 +55,7 @@ public static bool IsOnDispatcher()
             return CurrentDispatcher.CheckAccess();
         }
 
-        public static async void RunOnDispatcher(Action action)
+        public static async void RunOnDispatcher(Action action, bool runAsync = true)
```
Comment (rozele) [inline ReactWindows/ReactNative.Net46/Bridge/DispatcherHelpers.cs]: >bool runAsync = true [](start = 64, length = 20)

What is this default parameter used for?

File: ReactWindows/ReactNative.Net46/Bridge/DispatcherHelpers.cs [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Windows;
 using System.Windows.Threading;
```
Comment (rozele) [inline ReactWindows/ReactNative.Net46/Bridge/DispatcherHelpers.cs]: What is this using for?

File: ReactWindows/ReactNative.Net46/Shell/MainReactPackage.cs [hunk 3]
```diff
@@ -11,6 +11,7 @@
 using ReactNative.UIManager;
 using ReactNative.Views.Text;
 using ReactNative.Views.View;
+using ReactNative.Views.Scroll;
 using System;
```
Comment (rozele) [inline ReactWindows/ReactNative.Net46/Shell/MainReactPackage.cs]: sort.

File: ReactWindows/ReactNative.Shared/Bridge/ReactInstance.cs [hunk 4]
```diff
@@ -317,6 +317,7 @@ public NativeModulesReactCallback(ReactInstance parent)
             public void Invoke(int moduleId, int methodId, JArray parameters)
             {
                 _parent.QueueConfiguration.NativeModulesQueueThread.AssertOnThread();
+                var isOnDispatcher = _parent.QueueConfiguration.DispatcherQueueThread.IsOnThread();
```
Comment (rozele) [inline ReactWindows/ReactNative.Shared/Bridge/ReactInstance.cs]: Delete.

File: ReactWindows/ReactNative.Shared/DevSupport/DevSupportManager.cs [hunk 5]
```diff
@@ -10,6 +10,7 @@
 using System.Runtime.ExceptionServices;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Windows;
 #if WINDOWS_UWP
```
Comment (rozele) [inline ReactWindows/ReactNative.Shared/DevSupport/DevSupportManager.cs]: Delete.

File: ReactWindows/ReactNative.Shared.Tests/Modules/WebSocket/WebSocketModuleTests.cs [hunk 6]
```diff
@@ -4,6 +4,7 @@
 using ReactNative.Modules.Core;
 using ReactNative.Modules.WebSocket;
 using System.Threading;
+using System.Collections.Generic;
```
Comment (rozele) [inline ReactWindows/ReactNative.Shared.Tests/Modules/WebSocket/WebSocketModuleTests.cs]: Sort.

File: ReactWindows/ReactNative.Shared.Tests/Modules/WebSocket/WebSocketModuleTests.cs [hunk 7]
```diff
@@ -62,15 +63,22 @@ public void WebSocketModule_FailedEvent()
         {
             var waitHandle = new AutoResetEvent(false);
             var json = default(JObject);
+            bool websocketFailedCalled = false;
+
```
Comment (rozele) [inline ReactWindows/ReactNative.Shared.Tests/Modules/WebSocket/WebSocketModuleTests.cs]: I think Andrew reverted this file change except for the WaitOne timeouts.

**Comment by rozele** [comment]
:clock1:

**Comment by rozele** [comment]
You still have a conflict with the Examples submodule here.

---
In reply to: [260822835](https://github.com/ReactWindows/react-native-windows/pull/881#issuecomment-260822835) [](ancestors = 260822835)

**Comment by kevinvangelder** [comment]
Closing in favor of #894.

**Pull request closed without merging**